.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+While it might be obvious for you and experienced contributors, what and where `_typography.css` is; it is not obvious for newcomers. 
+
+If you already have refered to the full path earlier in the issue, it can be okay to refer to the file as `_typography.css`.
+
+_Example #2: writing reproducible steps for bug reports_
+
+The steps should be unambigously specific. This can be done for example by changing the step: 
+> 2. Type some text 
+
+to: 
+
+> 2. Type: 'text'
+ 
+There could be something with the _exact_ way you are doing it, that triggers the behaviour you are experiecing. 
+
+_Example #3: using code snippets to explain what and how_
+
+When reporting a bug or similar do not write something along the lines of: 
+> Whenever i use a `kirby-button` with attention level 1 on the frontpage, a warning is thrown
+
+Specify exactly where and how the button is inserted with a code snippet, along with exact details of the warning and where it can be found:
+> Inserting a `kirby-button` with attention level 1 as shown in the code-snippet, results in the following warning being logged in the browser console: `Warning goes here`.
+
+The latter is pure maintainer poetry. 
+
+These are but three examples of how issues can be specific and unambigious. But always keep in the back of your mind how you can be more specific and unambigious when writing issues. 
+
+#### The good issue follows the template 
+<!-- - Follow the templates! - they are as is to help you remember which information we're looking for -->
+Do not delete the template when creating an issue. Use it. 
+
+The issue templates exists to ease collaboration by providing structure and reminding you of which information other contributors are looking for. 
+They also provide helpful checklists that helps you follow [the process (INSERT PROPER LINK HERE)](#the-process) and lets others know how far you are. 
+
+If you experience that no template lives up to your specific need; well there is a good opportunity to contribute. Create an issue for a new template to be created! 
+
+#### The good issue has a good title
+<!-- - Spend some time on your titles 
+  - Make them alluring. 
+  - Short without too much detail 
+  - The issues are viewed in some sort of list view 80% of the time. -->
+As issues are viewed in some sort of list view 80% of the time (or so) the title is the part being read most often. 
+It can therefore be a good idea to spend some time formulating one. 
+
+A good title is short without too much detail. Details should be presented in the actual description. 
+
+If possible try to make the title compelling. While an open-source repository is not a newspaper where readers must be enganged at any cost, you still want to catch the interest of contributors. 
+
+The following title: 
+> Input causes error when you type over 50 characters and press enter 
+
+could be improved by writing: 
+> Input error on 50-character submit
+
```

Comment:
Suggestion: Sometimes it's helpful to write the issue description _before_ writing the title. It can be easier to formulate a brief but comprehensive title that summarises the issue. Consider adding a sentence or two about that.

Reply:
I've added the following in 3ec590cc23